lib/libnvpair/Makefile.am

```diff
@@ -31,10 +31,14 @@ libnvpair_la_LIBADD = \
 
 libnvpair_la_LIBADD += $(LIBTIRPC_LIBS) $(LTLIBINTL)
 
-libnvpair_la_LDFLAGS = -Wl,-z,defs
+libnvpair_la_LDFLAGS =
+
+if !ASAN_ENABLED
+libnvpair_la_LDFLAGS += -Wl,-z,defs
+endif
 
 if BUILD_FREEBSD
-libnvpair_la_LDFLAGS += -version-info 3:0:0
+libnvpair_la_LDFLAGS = -version-info 3:0:0
 else
-libnvpair_la_LDFLAGS += -version-info 1:1:0
+libnvpair_la_LDFLAGS = -version-info 1:1:0
```

Comment:
These need to still use `+=`.

Reply:
Yeah looks like that's left-over from the original patch.

Reply:
Done